concent_api/core/views.py

```diff
@@ -325,6 +326,7 @@ def handle_send_force_get_task_result(request, client_message: message.concents.
     validate_golem_message_task_to_compute(client_message.report_computed_task.task_to_compute)
     validate_id_value(client_message.report_computed_task.task_to_compute.compute_task_def['task_id'],      'task_id')
     validate_id_value(client_message.report_computed_task.task_to_compute.compute_task_def['subtask_id'],   'subtask_id')
+    force_get_task_result_acceptance_time = 2 * settings.CONCENT_MESSAGING_TIME + settings.MAXIMUM_DOWNLOAD_TIME
```

Comment:
Using `FORCE_ACCEPTANCE_TIME` in this use case was a mistake. Please don't name stuff after it.

I'd replace this with
``` python
force_get_task_result_deadline = (
    client_message.report_computed_task.task_to_compute.compute_task_def['deadline'] +
    2 * settings.CONCENT_MESSAGING_TIME +
    settings.MAXIMUM_DOWNLOAD_TIME
)
```